extensions/admin-pack/package.json

```diff
@@ -11,7 +11,8 @@
     "extensionPack": [
         "Microsoft.agent",
         "Microsoft.profiler",
-        "Microsoft.import"
+        "Microsoft.import",
+        "Microsoft.dacpac"
```

Comment:
no import extension bump, or that is happening elsewhere?

Reply:
import extension has been bumped by Kim when she was doing the extension spiting.